css/style.css

```diff
@@ -0,0 +1,221 @@
+@import url('https://fonts.googleapis.com/css2?family=Abhaya+Libre:wght@400;500;700&family=Roboto:wght@300;400;500;700&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    font-family: 'Roboto', sans-serif;
+    font-size: 14px;
+    font-weight: 400;
+}
+
+a {
+    color: #111111;
+    text-decoration: none;
+}
+
+.container {
+    max-width: 960px;
+    margin: 0px auto;
+}
+
+.postContain {
+    position: relative;
+    padding: 35px 0px 21px 40px;
+    background: #fefefe;
+}
+
+.postContain::after {
+    content: "";
+    width: 1px;
+    height: 100%;
+    position: absolute;
+    left: 0;
+    top: 0;
+    background: #6A6A6A;
+}
+
+.post {
+    margin-top: 30px;
+    display: flex;
+    flex-wrap: wrap;
+    box-shadow: 0px 2px 30px rgba(0, 0, 0, 0.03);
+    padding: 10px;
+    border-radius: 10px;
+}
+
+.postImg img {
+    width: 100%;
+    object-fit: cover;
+    border-radius: 7px;
+}
+
+.postImg {
+    width: 271px;
+    border-radius: 25px;
+}
+
+.postMain {
+    padding-left: 32px;
+    box-sizing: border-box;
+    flex-basis: 50%;
+    flex-grow: 1;
+}
+
+.postCategory {
+    margin-top: 0px;
+    padding: 5px 10px;
+    background: #262626;
+    display: inline-block;
+    color: #FBFBFB;
+    font-size: 11px;
+    margin-top: 10px;
+}
+
+.postTitle {
+    display: block;
+    margin-top: 13px;
+    font-size: 26px;
+    font-weight: 700;
+}
+
+.postDescription {
+    margin-top: 6px;
+    font-size: 17px;
+    color: #333333;
+    font-weight: 100;
+}
+
+.postInfoShare {
+    display: flex;
+    padding: 5px 0px;
+    margin-top: 10px;
+}
+
+.postInfo {
+    padding-right: 10px;
+    font-weight: 100;
+}
+
+.postInfo a {
+    font-weight: 700;
+}
+
+.postShare {
+    padding-left: 10px;
+    border-left: 1px solid #000000;
+    text-decoration: underline;
+    font-weight: 100;
+}
+
+.pagination {
+    margin-top: 30px;
+}
+
+.paginate {
+    display: flex;
+    list-style: none;
+}
+
+.paginate a {
+    display: block;
+    margin: 0px 5px;
+    font-size: 20px;
+}
+
+.paginateContain {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.arrowPaginate {
+    margin: 0px 10px;
+    cursor: pointer;
+    padding: 5px 8px;
+}
+
+.txtTitle {
+    color: #262626;
+    position: absolute;
+    left: -155px;
+    top: 10px;
+    font-size: 21px;
+}
+
+.txtTitle span {
+    letter-spacing: 0.15rem;
+}
+
+.arrowStories {
+    display: inline-block;
+    width: 25px;
+    height: 10px;
+}
+
+.arrowStories img {
+    width: 100%;
+    display: block;
+    transform: rotate(-90deg);
+}
+
+@media only screen and (max-width: 767px) {
```

Comment:
<img width="763" alt="Screen Shot 2020-09-09 at 11 16 40 PM" src="https://user-images.githubusercontent.com/56341874/92625532-f84c9f80-f2f2-11ea-8cd3-a2ec07ef21a7.png">
Consider reduce size of your mobile responsive. This is what happens when any phone has any resolution of width = 753px, you only see a part of the picture.

I think 576px is enough. Feel free to discuss in this conversation. ;;-;;

Reply:
Ok ok, I fixed.